src/components/Hero/Hero.jsx

```diff
@@ -0,0 +1,38 @@
+import React from "react";
+import "./Hero.scss";
+import { BrowserRouter as Router, Link } from "react-router-dom";
+import { Button } from "react-bootstrap";
+//Button is not working from react-bootstrap
+
+function Hero() {
+  return (
+    <div className="hero">
+      <div className="upperPart">
+        <div className="texts">
+          <h1>
+            Don’t need it?
```

Comment:
lets not use h1 here maybe an h2 would be better, since we need to only use h1 once in our website for SEO